src/app/components_ngrx/planner-list/ngx-datatables.less

```diff
@@ -0,0 +1,101 @@
+@import (reference) '../../../assets/stylesheets/_base';
+/// ngx-datatables css
+
+.cell-class {
+  overflow: visible !important;
+  align-items: center;
+  height: 100%;
```

Comment:
> cell items can be impacted if we don't set it to center and height 100% makes sure any item takes full height to keep everything vertically centered

```align-items: center``` is only working with flexbox and ```cell-class``` is not a flexbox. And we are already setting cell height here https://github.com/fabric8-ui/fabric8-planner/blob/master/src/app/components_ngrx/planner-list/planner-list.component.html#L102.